api/registrations/serializers.py

```diff
@@ -345,6 +337,21 @@ def get_view_only_links_count(self, obj):
     def get_total_comments_count(self, obj):
         return obj.comment_set.filter(page='node', is_deleted=False).count()
 
+    def anonymize_registered_meta(self, obj):
+        """
+        Looks at every question on every page of the schema, for any titles
+        matching ANONYMIZED_TITLES.  If present, deletes that question's response
+        from meta_values.
+        """
+        meta_values = obj.registered_meta.values()[0]
+        if is_anonymized(self.context['request']):
+            registration_schema = RegistrationSchema.objects.get(_id=obj.registered_schema_id)
+            for page in registration_schema.schema['pages']:
+                for question in page['questions']:
+                    if question['title'] in ANONYMIZED_TITLES and meta_values.get(question.get('qid')):
+                        del meta_values[question['qid']]
+        return meta_values
```

Comment:
Currently the `Author` response is the only one we're deleting - this is question 2 in the Registered Report and Prereg schema.  I'm just deleting the question response, but I can also just make the value an empty string? Either way.

Reply:
This seems good and matches existing functionality. I could see a future where we'd just put in something like "This information has been anonymized" or whatever, but I think that's a future when we have a lot more free time than we do now.